artifacts/flagger-istio-mtls/1-flagger-rules.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: flagger-namespace
+  namespace: flagger
+spec:
+  host: "*.flagger.svc.cluster.local"
+  trafficPolicy:
+    tls:
+      mode: DISABLE
```

Comment:
Instead of adding all this yaml to the repo, I think is better to just put the Destination rule and policy in the Load testing section in docs and explain what it does.